README.md

```diff
@@ -1 +1,63 @@
-# terraform-alicloud-cdn-domain
\ No newline at end of file
+Alicloud CDN Terraform Module
+terraform-alicloud-cdn-domain
+=============================================
+
+Terraform module which creates CDN resources on Alibaba Cloud.
+
+These types of resources are supported:
+
+* [CDN](https://www.terraform.io/docs/providers/alicloud/r/cdn_domain.html)
+
+Root module calls these modules which can also be used separately to create independent resources:
+
+* [cdn](https://github.com/alibaba/terraform-alicloud-cdn/tree/master/modules/cdn) - create cdn domain
+
+
+
+Usage
+-----
+You can use this in your terraform template with the following steps.
+
+1. Adding a module resource to your template, e.g. main.tf
+
+
+         module "cdn" {
+            source = "terraform-alicloud-modules/cdn-domain/alicloud"
+
+            domain_name = "${var.domain_name}"
+            type    = "${var.type}"
+
+            origin_type = "${var.origin_type}"
+            origins     = "${var.origins}"
+         }
+
+2. Setting values for the following variables through environment variables:
+
+    - ALICLOUD_ACCESS_KEY
+    - ALICLOUD_SECRET_KEY
+
+
+## Inputs
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| domain_name | The name of a new cdn domain | string | no | yes |
+| type | The is cdn business type | string | web | yes |
+| origin_type | The type origin type | string | domain | yes |
+| origin_port | The type origin port | string | 80 | yes |
+| origins     | The origin instanes | list | no | yes |
```

Comment:
What do the `The is cdn business type`, `The type origin type` and `The type origin port` mean? Can you clear them?